### PR TITLE
docs(fix): Update API arguments

### PR DESCRIPTION
**Pull request by bucanero**
Docs Preview: [Dgraph Preview](https://)

### CODE REVIEW

File: content/cloud-api/backend.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ mutation CreateDeployment($dep: NewDeployment!) {
 
 **Arguments**
 
-* `deployment`: parameter object for new deployment
+* `newDeployment`: parameter object for new deployment
   * `name`: name of the deployment
```
Comment (verneleem) [inline content/cloud-api/backend.md:116]: We don't do the type dotted notation here, but we do it on the same page below in the updateDeploymentInput. We need consistency one way or the other

File: content/cloud-api/schema.md [hunk 2]
```diff
@@ -98,7 +98,7 @@ https://${DEPLOYMENT_URL}/admin
 
 ```graphql
 mutation($sch: String!) {
-  updateGQLSchema(input: { set: { schema: $sch } }) {
+  updateGQLSchema(input: { set: { schema: $schema } }) {
```
Comment (verneleem) [inline content/cloud-api/schema.md:101]: This also needs changed on line 100 above.

File: content/cloud-api/lambda.md [hunk 3]
```diff
@@ -79,17 +79,17 @@ https://cerebro.cloud.dgraph.io/graphql
 ### API Command
 
 ```graphql
-query GetLambdaLogs($input: LambdaLogsInput!) {
-  getLambdaLogs(input: $input)
+query GetLambdaLogs($lambdaLogsInput: LambdaLogsInput!) {
```
Comment (verneleem) [inline content/cloud-api/lambda.md:82]: similar changes needs done on lines 138-147 and 204-213

**Review by verneleem — CHANGES_REQUESTED** [review]

File: content/cloud-api/lambda.md (hunk 3 above)
Comment (bucanero) [reply content/cloud-api/lambda.md:82]: fixed

File: content/cloud-api/schema.md (hunk 2 above)
Comment (bucanero) [reply content/cloud-api/schema.md:101]: fixed

File: content/cloud-api/backend.md (hunk 1 above)
Comment (bucanero) [reply content/cloud-api/backend.md]: fixed

**Pull request merged**